packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -104,12 +104,16 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
    * Need to expose this method in case of popups since focus needs to be set when popup is opened
    */
   public focus() {
-    let { firstFocusableSelector } = this.props;
+    const { firstFocusableSelector } = this.props;
+    const focusSelector = typeof firstFocusableSelector === 'string'
+      ? firstFocusableSelector
+      : firstFocusableSelector && firstFocusableSelector();
+
     let _firstFocusableChild;
     let root = this.refs.root;
 
-    if (firstFocusableSelector) {
-      _firstFocusableChild = root.querySelector('.' + firstFocusableSelector);
+    if (focusSelector) {
+      _firstFocusableChild = root.querySelector('.' + focusSelector);
```

Comment:
@dzearing here is that assumed class we spoke about yesterday. Do we want to deal with this '.' now?

Reply:
yes please, kill it

Reply:
@antonlabunets @zoarif This may affect some of your scenarios (FocusTrapZone.firstFocusableSelector should accept a selector, not a classname.)

Reply:
Are you asking me to change the PR? :)

Reply:
Yes I'd recommend removing it.

This logic is wrong. The property is called "selector" rather than "className", so it should behave as a selector.

Reply:
That would be an API breaking change, right?